docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+# BSC Staking Overview
+
+BNB Smart Chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain where staking plays a crucial role.
+[BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces native staking on BSC.
+This section covers the fundamental staking concepts and operations on BSC.
+
+## Basic Concepts
+
+### Consensus Engine
+
+BSC uses a consensus mechanism which combines DPoS and PoA for consensus, so that:
+
+* Blocks are produced by a limited set of validators.
+* Validators take turns to produce blocks in a PoA manner.
+* Validator set are elected in and out based on a staking based governance.
+
+To determinate which validators are eligible to produce blocks, that's why staking mechanism is needed.
+
+### Validator Set
+
+The validator set is the group of nodes that are responsible for validating transactions and producing blocks on the
+BSC. The validator set is determined by the amount of staking credit each validator has, which reflects the amount of
+BNB staked by the validator and its delegators. The top validators with the most staking credit are selected as the
```

Comment:
reader dont know `staking credit` yet, just use `staking`

Reply:
Fixed here, but will remain in the following parts, because there is some explanation about credit, and if no credit we can not explain how staking works.
<img width="1264" alt="image" src="https://github.com/huihzhao/docsite/assets/90749943/c868d507-6531-4cb4-9485-bef61671443e">